shell/client/shell.html

```diff
@@ -186,6 +190,39 @@
   {{/with}}
 </template>
 
+<template name="requestAccess">
+  <div class="grain-interstitial">
+  <p>You do not have permission to access this grain.</p>
+  {{#if currentUser}}
+    {{#with status}}
+      {{#if showButton}}
+        <button class="request-access" title="Request access">
+          Request Access
+        </button>
+      {{/if}}
+      {{#if chooseIdentity}}
+        <p>Requesting access will reveal your identity to the grain owner.</p>
+        <p> {{chooseIdentityText}} </p>
+        {{> identityPicker identityPickerData}}
+      {{/if}}
+      {{#if waiting}}
+         <p>Sending your request...</p>
+      {{/if}}
+      {{#if success}}
+        <p>
+          Your request has been sent. You will receive an email if your request gets approved.
```

Comment:
"gets approved" sounds ever so slightly weird to me. I feel like "is approved" would sound less weird. This may just be me.